share/sysbuild/image_configurations/BOOTLOADER_image_default.cmake

```diff
@@ -12,6 +12,7 @@ set(keytypes CONFIG_BOOT_SIGNATURE_TYPE_NONE
 
 if(SB_CONFIG_BOOT_SIGNATURE_TYPE_NONE)
   set(keytype CONFIG_BOOT_SIGNATURE_TYPE_NONE)
+  set_config_bool(${ZCMAKE_APPLICATION} CONFIG_MBEDTLS n)
```

Comment:
> Fixes an issue whereby a project is configured then the configuration is changed multiple times to work with signatures to without whereby a conflict between tinycrypt and MBEDTLS would appear

How can we reproduce this?

Reply:
Configure smp_svr using sysbuild, run `ninja sysbuild_menuconfig` and change signature type to none, then build

Reply:
Thanks, I'm not sure why I couldn't reproduce it at all before. But now I can see that this is not a sysbuild-specific issue, because I can: 

1. Run `cmake -S$(west topdir)/bootloader/mcuboot/boot/zephyr -GNinja -DBOARD=nrf52840dk_nrf52840`
2. Set `CONFIG_BOOT_SIGNATURE_TYPE_NONE=y`
3. Build

and get the same error. So, shouldn't this be fixed in MCUboot proper?

Reply:
It can't, mbedlts will default to on with the initial config, when `menuconfig` is ran and the setting is changed, it is no longer using the `default` value but the saved value from the .config file, so it is not able to know that it can be un-selected.

Reply:
How about selecting `MBEDTLS_PROMPTLESS` in MCUboot?

Reply:
Have updated